src/main/java/com/cleanroommc/groovyscript/compat/mods/prodigytech/SimpleRecipeHandler.java

```diff
@@ -0,0 +1,109 @@
+package com.cleanroommc.groovyscript.compat.mods.prodigytech;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.IIngredient;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.helper.ingredient.OreDictIngredient;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import lykrast.prodigytech.common.recipe.MagneticReassemblerManager;
+import lykrast.prodigytech.common.recipe.RotaryGrinderManager;
+import lykrast.prodigytech.common.recipe.SimpleRecipe;
+import lykrast.prodigytech.common.recipe.SimpleRecipeManager;
+import lykrast.prodigytech.common.util.Config;
+import net.minecraft.item.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+public abstract class SimpleRecipeHandler extends SimpleRecipeHandlerAbstract<SimpleRecipe> {
+    @RecipeBuilderDescription(example = {
+        @Example(".input(item('minecraft:gold_ingot')).output(item('minecraft:diamond')).time(50)"),
+        @Example(".input(item('minecraft:iron_ingot')).output(item('minecraft:coal'))")
+    })
+    public SimpleRecipeHandler.RecipeBuilder recipeBuilder() {
+        return new SimpleRecipeHandler.RecipeBuilder();
+    }
+
+    SimpleRecipeHandler(String name, SimpleRecipeManager instance) {
+        super(name, instance);
+    }
+
+    @Property(property = "input", valid = @Comp("1"))
+    @Property(property = "output", valid = @Comp("1"))
+    public class RecipeBuilder extends AbstractRecipeBuilder<SimpleRecipe> {
+        @Property(valid = @Comp(value = "1", type = Comp.Type.GTE))
+        private int time = -1;

```

Comment:
why not set it to `getDefaultTime()` by default?

Reply:
I should do it, it caused mental headaches for me when writing it since I mainly use Python and the semantics of that would be meaningfully different